module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -1277,9 +1272,9 @@ public function editlistAction()
 
         $listTags = null;
         if ($this->listTagsEnabled() && !$newList) {
-            $tagService = $this->getDbService(TagServiceInterface::class);
+            $tagsService = $this->getDbService(TagsService::class);
```

Comment:
Shouldn't this be `$this->serviceLocator->get(TagsService::class)`?

Reply:
Apparently yes, though not sure how that happened. Sorry!

Reply:
No problem; sorry for wasting your time by making such a silly mistake in the first place. ;-)